### DISCUSSION TITLE
git-lfs support?

### DISCUSSION THREAD

**Discussion by erinok** [jj-vcs/jj#77] [Q&A] [answered]
hi all, love the idea of this! seems like a way better workflow than git.

unfortunately, the project i'm currently working on makes heavy use of git-lfs. experimentally, it seems like jj doesn't support that yet.

is that correct? if so, are there plans to add it eventually, or out of scope?

(thx!)

**Comment by martinvonz** [comment, marked as answer]
Yes, that's correct, there's no support for Git LFS yet. I created #80 for it so we can track work in it there and so people can vote for it.

**Comment by erinok** [comment]
Sounds good, thank you!
